docs/tutorials/data-modeling-lifecycle/part-2-analytic-solution.md

```diff
@@ -1 +1,169 @@
+the power on the market.
+
+## Selecting from the Enterprise Model
+The most important part of Olav's new solution model is the `WindTurbine` and the `WindFarm` concepts. First,
+let's look at the `WindTurbine` from the enterprise model.
+
+| Class       | Property             | Value Type |
+|-------------|----------------------|------------|
+| WindTurbine | name                 | string     |
+| WindTurbine | type                 | string     |
+| WindTurbine | geoLocation          | Point      |
+| WindTurbine | manufacturer         | string     |
+| WindTurbine | lifeExpectancy       | integer    |
+| WindTurbine | ratedPower           | float      |
+| WindTurbine | hubHeight            | float      |
+| WindTurbine | actualPower          | timeseries |
+| WindTurbine | arrayCableConnection | ArrayCable |
+
+The most important properties for Olav's forecasting model are the `actualPower` and the `ratedPower`. In addition,
+he needs the `geoLocation` to calculate the wind speed and the `hubHeight` to adjust the wind speed to the height
+of the wind turbine. In addition, there might be differences in the power output between the wind turbines based
+on the manufacturer, life expectancy, and the type of wind turbine. However, Olav decides that
+arrayCableConnection is not important for his forecasting model, so the `WindTurbine`in the solution model will
+not have this property.
+
+Next, let's look at the `WindFarm` from the enterprise model.
+
+| Class       | Property             | Value Type         |
+|-------------|----------------------|--------------------|
+| WindFarm    | name                 | string             |
+| WindFarm    | geoLocation          | Polygon            |
+| WindFarm    | ratedPower           | float              |
+| WindFarm    | activePower          | timeseries         |
+| WindFarm    | substation           | OffshoreSubstation |
+| WindFarm    | exportCable          | ExportCable        |
+| WindFarm    | arrayCable           | ArrayCable         |
+| WindFarm    | windTurbines         | WindTurbine        |
+
+Olav decides that this can be simplified for his forecasting model down to `name`, `geoLocation` and `windTurbines`.
+The `ratedPower` and `activePower` are not needed as it is the sum for all the `WindTurbines`. The
+`substation`, `exportCable`, and `arrayCable` are not needed for the forecasting model.
+
+Based on the choices above, Olav will also include `Point` and `Polygon` from the enterprise model as they are needed
+for the `geoLocation` of the `WindTurbine` and `WindFarm`.
+
+## Adding new Concepts
+
+To make good forecasts, Olav needs historical and forecasted weather data. Thus, he will have to extend the solution
+model with new concepts for the weather data. In addition, he needs to add a concept for the forecasted power output
+of the wind turbines.
+
+Olav chose to model the weather data as a `WeatherLocation`. The `WeatherLocation` concept will look
```

Comment:
Call this WeatherStation instead of WeatherLocation

Reply:
Nice, I struggled with that name.